jobs/sql/extractions/salud_mental_encuentro_staging.sql

```diff
@@ -1,1025 +1,331 @@
+update salud_mental_encountero set PHQ9_q3 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13663');
+update salud_mental_encountero set PHQ9_q4 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13664');
+update salud_mental_encountero set PHQ9_q5 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13665');
+update salud_mental_encountero set PHQ9_q6 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13666');
+update salud_mental_encountero set PHQ9_q7 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13667');
+update salud_mental_encountero set PHQ9_q8 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13668');
+update salud_mental_encountero set PHQ9_q9 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13669');
+
+update salud_mental_encountero set GAD7_q1 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13671');
+update salud_mental_encountero set GAD7_q2 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13672');
+update salud_mental_encountero set GAD7_q3 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13673');
+update salud_mental_encountero set GAD7_q4 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13674');
+update salud_mental_encountero set GAD7_q5 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13675');
+update salud_mental_encountero set GAD7_q6 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13676');
+update salud_mental_encountero set GAD7_q7 = obs_value_numeric_from_temp(encounter_id, 'PIH', '13677');
+
+update salud_mental_encountero set PHQ9_score = obs_value_numeric_from_temp(encounter_id, 'PIH', '11586');
+update salud_mental_encountero set GAD7_score = obs_value_numeric_from_temp(encounter_id, 'PIH', '11733');
+
+-- ############## Observations ###################################
+
+update salud_mental_encountero set visit_Reason = obs_value_coded_list_from_temp(encounter_id, 'PIH', '6189', 'en');
+update salud_mental_encountero set analysis_notes = obs_value_text_from_temp(encounter_id, 'PIH', '1364');
+update salud_mental_encountero set case_notes = obs_value_text_from_temp(encounter_id, 'PIH', '1336');
+update salud_mental_encountero set Treatment_plan = obs_value_text_from_temp(encounter_id, 'PIH', '10534');
+update salud_mental_encountero set lab_tests_ordered = obs_value_text_from_temp(encounter_id, 'PIH', '11762');
+update salud_mental_encountero set next_appointment = date(obs_value_datetime_from_temp(encounter_id, 'PIH', '5096'));
+
+-- ############# Medications #####################################
+
+-- First thing is to get all medication obs name obs in order.
+
+select concept_from_mapping('PIH','1282') into @medName;
+
+drop table if exists temp_mh_medication_name_obs;
+create temporary table temp_mh_medication_name_obs
+select encounter_id,
+       obs_id,
+       value_drug,
+       @obs_group_id:=obs_group_id obs_group_id,
+       @row_number:=if(@obs_group_id = obs_group_id, @row_number + 1, 1) obs_group_order
```

Comment:
This row needs to be first then the previous row to come second

Reply:
Makes sense, can you change that?